ios/RNTPTDocumentView.m

```diff
@@ -1873,6 +1872,60 @@ - (void)applyLayoutMode:(PTPDFViewCtrl *)pdfViewCtrl
     }
 }
 
+- (void)setUrlExtraction:(BOOL)urlExtraction
+{
+    [self.documentViewController.pdfViewCtrl SetUrlExtraction:urlExtraction];
+}
+
+- (void)setPageBorderVisibility:(BOOL)pageBorderVisibility
+{
+    PTPDFViewCtrl *pdfViewCtrl = self.documentViewController.pdfViewCtrl;
+    [pdfViewCtrl SetPageBorderVisibility:pageBorderVisibility];
+    [pdfViewCtrl Update:YES];
+}
+
+- (void)setPageTransparencyGrid:(BOOL)pageTransparencyGrid
+{
+    PTPDFViewCtrl *pdfViewCtrl = self.documentViewController.pdfViewCtrl;
+    [pdfViewCtrl SetPageTransparencyGrid:pageTransparencyGrid];
+    [pdfViewCtrl Update:YES];
+}
+
+- (void)setDefaultPageColor:(NSDictionary *)defaultPageColor
+{
+    if (defaultPageColor) {
+        NSArray *keyList = defaultPageColor.allKeys;
```

Comment:
```suggestion
        NSArray *keyList = defaultPageColor.allKeys;
        NSAssert([keyList containsObject:PTColorRedKey] &&
            [keyList containsObject:PTColorGreenKey] &&
            [keyList containsObject:PTColorBlueKey]);
```

Reply:
If I understand correctly, NSAssert would crash the app if condition is not true. So why is it better than an if check in this case?

Reply:
Assert will "crash" the app, but only when debugging. Was I correct that not having one of these keys would constitute a programming error? If so, it is nice to alert the programmer in a way that's impossible to miss. But in release we can fail in a gentler way.